src/core/js/models/adaptModel.js

```diff
@@ -739,8 +747,20 @@ define([
 
         onIsComplete: function() {
             this.checkCompletionStatus();
-
             this.checkLocking();
+        },
+
+        onAll: function(type, model, value) {
+            if (!_.contains(this.bubblingEvents, type)) return;
+            var event = new ModelEvent(type, model, value);
+            this.bubble(event);
+        },
+
+        bubble: function(event) {
+            if (!event.bubbles) return;
+            event.addPath(this);
+            this.trigger("bubble:"+event.type, event);
+            this.trigger("bubble", event);
```

Comment:
could maybe merge these two lines into one 
```js
this.trigger('bubble bubble:' + event.type, event);
```

Reply:
I want `bubble` to call after the more specific `bubble:type`, so imma invert your string but yes.